cmd/commands/service/command.go

```diff
@@ -111,6 +111,11 @@ func (sc *serviceCommand) Run(ctx *cli.Context) (err error) {
 		serviceTypes = strings.Split(ctx.Args().Get(0), ",")
 	}
 
+	if len(serviceTypes) == 0 {
```

Comment:
This leaves deadcode on line 109.
Just revert PR #2424

Reply:
It does not leave it dead if its not a 0